TgeeT/Questioner
# Feature  #163046371 creating the rsvp endpoint

#### What does this PR do?

Create an endpoint to rsvp a meetup
#### Description of Task to be completed?

write failing test for the rsvp
write the bear minimum code to pass the tests

#### How should this be manually tested?

follow the steps below:
1. git clone https://github.com/tgeet/Questioner.git
2. git checkout to ft-rsvp-163046371 branch
3. flask run the app and use postman to verify

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163046371
